packages/shopware-6-client/src/apiService.ts

```diff
@@ -1,11 +1,11 @@
-import axios from "axios";
-import https from "https";
+import axios, { AxiosInstance } from "axios";
+import { Agent } from "https";
 import { config } from "./settings";
 import { responseInterceptor, errorInterceptor } from "./interceptors";
 
-export const apiService = axios.create({
+export const apiService: AxiosInstance = axios.create({
   // temporary fix to prevent TLS issues
-  httpsAgent: new https.Agent({
+  httpsAgent: new Agent({
     rejectUnauthorized: false,
   }),
```

Comment:
can we get rid of it here? I know what's for, but since we have no problems with certificates and connection between layers - we can go without it.

Reply:
we'll do it in another PR

Reply:
I've created issue to remove that ;)